src/content/blog/2024/04/25/react-19-upgrade-guide.md

```diff
@@ -97,6 +98,36 @@ If you're using TypeScript, you also need to update the types. Once React 19 is
 
 We're also including a codemod for the most common replacements. See [TypeScript changes](#typescript-changes) below.
 
+## Codemods {/*codemods*/}
+
+To help with the upgrade, we've worked with the team at [codemod.com](https://codemod.com) to publish codemods that will automatically update your code to many of the new APIs and patterns in React 19.
+
+The Codemod team have published all codemods to [`react-codemod`](https://github.com/reactjs/react-codemod) and will be helping maintain the React codemods moving forward, but we recommend using the `codemod` command for a better experience. 
```

Comment:
this sentence is a little hard to parse for me, what does it mean to use the `codemod` command (as opposed to?)

Reply:
Will update with a better blurb